source/common/common/utility.cc

```diff
@@ -67,6 +71,65 @@ bool StringUtil::atoul(const char* str, uint64_t& out, int base) {
   }
 }
 
+absl::string_view StringUtil::ltrim(absl::string_view source) {
+  const absl::string_view::size_type pos = source.find_first_not_of(WhitespaceChars);
+  if (pos != absl::string_view::npos) {
+    source.remove_prefix(pos);
+  } else {
+    source.remove_prefix(source.size());
+  }
+  return source;
+}
+
+absl::string_view StringUtil::rtrim(absl::string_view source) {
+  const absl::string_view::size_type pos = source.find_last_not_of(WhitespaceChars);
+  if (pos != absl::string_view::npos) {
+    source.remove_suffix(source.size() - source.find_last_not_of(WhitespaceChars) - 1);
```

Comment:
nit: Does `find_last_not_of` need to be computed again? Can't you just use `pos`?

Reply:
oops.. sorry, bad fix!